charts/graphscope-store/templates/_helpers.tpl

```diff
@@ -70,12 +70,12 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 Return the proper graphscope-store image name
 */}}
 {{- define "graphscope-store.image" -}}
-{{ include "graphscope-store.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
+{{ include "graphscope-store.images.image" (dict "imageRoot" .Values.image "global" .Values.global "DefaultTag" .Chart.AppVersion ) }}
```

Comment:
@siyuan0322 The new kv is contained in the dict, and I think the `imageRoot.DefaultTag` points to the `.Chart.AppVersion` of the `Chart.yaml`. Is there something I missed?

In fact, we can get the following result.
```shell
$ helm install graphscope-store "./graphscope-store"
$ kubectl get po coordinator-graphscope-6d8fccbb9c-cl6vm -oyaml | grep image
    - --k8s_gs_image
    - --k8s_etcd_image
    - --k8s_image_pull_policy
    - --k8s_image_pull_secrets
    image: registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:0.18.0
    imagePullPolicy: Always
  - image: registry.cn-hongkong.aliyuncs.com/graphscope/jupyter:0.18.0
    imagePullPolicy: Always
```

Reply:
Thanks for the catching, I have tried again and find the tag is exactly `latest`.

```shell
$ kubectl get po graphscope-store-coordinator-0 -n gh-store -oyaml | grep image
            f:image: {}
            f:imagePullPolicy: {}
    image: registry.cn-hongkong.aliyuncs.com/graphscope/graphscope-store:latest
    imagePullPolicy: IfNotPresent
    image: registry.cn-hongkong.aliyuncs.com/graphscope/graphscope-store:latest
    imageID: registry.cn-hongkong.aliyuncs.com/graphscope/graphscope-store@sha256:b00a5a15a48a5bca86d02f7c16a07335377bc3e617b3d67c22331b5a5ba9008a
```

I'll fix it ASAP.